grpc_unary_attestation/src/client.rs

```diff
@@ -63,16 +66,23 @@ pub struct AttestationClient {
 }
 
 impl AttestationClient {
-    pub async fn create(
+    /// Create an [`AttestationClient`] with a [`PlaceholderAmdAttestationVerifier`].
+    pub async fn create(uri: &str) -> anyhow::Result<Self> {
+        Self::create_with_attestation_verifier(uri, PlaceholderAmdAttestationVerifier).await
```

Comment:
Will this placeholder later become an argument for `create` (once we have attestation)?
Or it will be hardcoded?

Reply:
ah, I see the next function now